### PR TITLE
Publish to NPM

### PR DESCRIPTION
**Pull request by offerijns**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,3 +23,9 @@ jobs:
   #     - dapp testnet &
   #     - ./bin/deploy.sh
   #     - npm run test
+  - stage: Publish patch
+    if: "(NOT type IN (pull_request)) AND (branch = master)"
+    script:
+      - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
```
Comment (philipstanislaus) [inline .travis.yml:29]: Do we really want to redirect stderr?

File: .travis.yml [hunk 2]
```diff
@@ -23,3 +23,9 @@ jobs:
...
+      - npm run publish:patch
```
Comment (philipstanislaus) [inline .travis.yml:30]: Can we actually do the same here as we do on axis, i. e. publish canary versions only for now? 

I think we should keep npm as unstable until rev pools are done.

**Review by philipstanislaus — CHANGES_REQUESTED** [review]
Left a few comments, thanks!

File: .travis.yml (hunk 1 above)
Comment (offerijns) [reply .travis.yml:29]: I copied this from the way it's done in axis. Travis CI isn't setup yet (I don't have access, have requested this from Lucas). So I'd suggest we keep it like this for now and I revise/check this once Travis CI is set up.

File: .travis.yml (hunk 2 above)
Comment (offerijns) [reply .travis.yml:30]: I couldn't figure out how to do that with npm version. Lerna has built in functionality for this, but npm version doesn't appear to. I'll double check this.

Comment (offerijns) [reply .travis.yml]: This is now fixed!

File: .travis.yml (hunk 1 above)
Comment (philipstanislaus) [reply .travis.yml:29]: Sounds good, thanks!

**Review by philipstanislaus — APPROVED** [review]
🎉

**Pull request merged**